SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/HoloNetViewModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Webhook;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition.ViewModel
+{
+    public class HoloNetViewModel : GuiViewModelBase<HoloNetViewModel, GuiPayloadBase>
+    {
+        public string HoloNetText
+        {
+            get => Get<string>();
+            set => Set(value);
+        }
+
+        public const int MaxHoloNetTextLength = 800;
+
+        protected override void Initialize(GuiPayloadBase initialPayload)
+        {
+            HoloNetText = string.Empty;
+            WatchOnClient(model => model.HoloNetText);
+        }
+
+        public Action OnClickSubmit() => () =>
+        {
+            var price = 2500;
```

Comment:
On line 18 you have a constant to define the text length. Please do the same thing for the price so it's clear when you first open the file what it's set to. The price currently doesn't change so there's no reason to make it a variable.

Place this below the holonet text:

private const int Price = 2500;

Reply:
Resolved. Change price to 2500 too!